spec/controllers/social_networking/profiles_controller_spec.rb

```diff
@@ -3,26 +3,25 @@
 module SocialNetworking
   describe ProfilesController, type: :controller do
     context "participant is authenticated" do
-      let(:participant) { double("participant") }
-      let(:profile) { double("profile") }
+      let(:participant) { instance_double("participant") }
```

Comment:
Regarding our earlier conversation, should this be `instance_double(Participant)` instead?